templates/next/components/foo/Form.tsx

```diff
@@ -91,7 +98,7 @@ export const Form: FunctionComponent<Props> = ({ {{{lc}}} }) => {
                 onBlur={handleBlur}
               />
             </div>
-            { errors.{{name}} && touched.{{name}} && <div className="invalid-feedback">{ errors.{{name}} }</div> }
+            { errors.{{name}} && touched.{{name}} && <div className="text-danger">{ errors.{{name}} }</div> }
```

Comment:
`invalid-feedback` looks best suited: https://getbootstrap.com/docs/4.1/components/forms/

Reply:
I changed for "text-danger" because "invalid-feedback" did'nt show (see: https://stackoverflow.com/questions/45673354/bootstrap-4-invalid-feedback-with-input-group-not-displaying), but I'll try another solution with "invalid-feedback" like so (https://stackoverflow.com/questions/50431450/force-to-show-invalid-feedback-in-bootstrap-4)